layouts/shortcodes/page-adopters.html

```diff
@@ -7,23 +7,14 @@
     Christopher Guindon <chris.guindon@eclipse-foundation.org>
     
   SPDX-License-Identifier: EPL-2.0
---> 
+-->
 <div class="row">
-{{ range sort .Site.Data.adopters.adopters.projects "name" "asc"}}
-  <div class="col-md-24 adopters">
-    <h2 id="{{ .id }}">
-      {{ .name }} 
-      <a class=" btn btn-xs btn-secondary" href="https://projects.eclipse.org/projects/{{ .id }}">{{ .id }}</a>
```

Comment:
Do we want to drop this button? 🤔

Reply:
This would be an enhancement on the project-adopters JS rather than here. I don't think there's any harm with providing the button in the implementation though